src/el-data-table.vue

```diff
@@ -636,6 +637,8 @@ export default {
           ''
         )
 
+      this.params = params
```

Comment:
this code is worst,  cause side effects and next function rely on it. 
don't like this code style